pyfolio/tears.py

```diff
@@ -81,7 +81,7 @@ def create_full_tear_sheet(returns,
                            factor_returns=None,
                            factor_loadings=None,
                            pos_in_dollars=True,
-                           return_fig=False):
```

Comment:
is that right?

Reply:
I think so, for some reason we were accepting this argument but it wasn't being used anymore. return_fig is only used for the create_tear_sheet functions that produce one figure with multiple plots, like the returns tear sheet (the full tear sheet produces a bunch of figures). I am surprised this got by flake8 in the past.